cmd/internal/cli/startvm_darwin.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+)
+
+func startVM(sifImage, singAction, cliExtra string, isInternal bool) error {
```

Comment:
Eventually `startVM` [for all archs] should be moved to `internal/app/singularity`, but for now this is fine.